### PR TITLE
Port /contact/ pages to Protocol (Fixes #8506)

### LINKED ISSUE #8506: Port /contact/ pages to Protocol
## Description

The /contact/ pages are still using Pebbles and need porting to Protocol.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Pages are updated in production to use Protocol.

#### Tasks
- [x] Update templates and scss.
- [x] Remove any stray assets.
- [x] Update tests?

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Ports /contact/ pages to Protocol

## Issue / Bugzilla link
#8506

## Testing
Successful test run: https://gitlab.com/mozmeao/www-config/pipelines/117259206

### CODE REVIEW

File: media/css/mozorg/contact-spaces.scss [hunk 1]
```diff
@@ -16,86 +20,42 @@
...
-}
-
-a:hover,
-a:active,
-a:focus {
-    color: darken($color-link, 10%);
-}
-
-// Add a double-arrow after 'go' links
-.go {
```
Comment (amychurchwell) [inline media/css/mozorg/contact-spaces.scss]: nit: `class="go"` could be removed from a lot of templates in contact/communities.

File: media/css/mozorg/contact-spaces.scss [hunk 2]
```diff
@@ -104,321 +64,123 @@ html[dir="rtl"] {
...
 .page-content {
+    padding-top: $layout-md;
     position: relative;
 }
 
+.mzp-has-sidebar {
+    @include clearfix;
+}
+
 .category-tabs {
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply media/css/mozorg/contact-spaces.scss]: I'm going to resist more radical redesigns of this page for now. It really needs a complete overhaul. Using the navigation menu is definitely a possibility, but it doesn't give a clear sense over which tab is active.

File: bedrock/mozorg/templates/mozorg/contact/contact-base.html [hunk 3]
```diff
@@ -4,20 +4,19 @@
...
   {{ css_bundle('contact-spaces') }}
 {% endblock %}
 
 {% block content %}
 
 <main id="main-content">
-  <div class="content">
+  <div class="mzp-l-content">
     <header class="page-head">
       <h1 class="page-title"><span>{{ _('Where is Mozilla?') }}</span></h1>
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/mozorg/templates/mozorg/contact/contact-base.html]: Fixed, thanks!

**Comment by alexgibson** [comment]
Thanks for the review @amychurchwell, updated!

**Pull request merged**